src/material/chips/chip-list.ts

```diff
@@ -455,7 +451,11 @@ export class MatChipList
    * @docs-private
    */
   setDescribedByIds(ids: string[]) {
-    this._ariaDescribedby = ids.join(' ');
+    if (ids.length) {
+      this._elementRef.nativeElement.setAttribute('aria-describedby', ids.join(' '));
+    } else {
+      this._elementRef.nativeElement.removeAttribute('aria-describedby');
```

Comment:
Since this is very similar to https://github.com/angular/components/pull/19587 and about making it in sync with the input, it would seem reasonable adding the `useAriaDescribedBy` code as well (allowing for additional user description). Would you be able to add that as well?

Reply:
Haha, yes, it's copied straight from your PR. :) Sure, I can add that in a bit.

Reply:
@devversion Updated.

Reply:
awesome, thank you!